components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -82,6 +83,20 @@ static CGFloat randomNumberNotEqualTo(const CGFloat targetNumber) {
   return [string copy];
 }
 
+static const CGFloat MDCButtonTestingSubclassRadius = 9;
+
+/*
+ A simple testing subclass used to verify the behavior of mdc_layerCornerRadius.
+ */
+@interface MDCButtonTestingSubclass : MDCButton
+@end
+
+@implementation MDCButtonTestingSubclass
```

Comment:
Minor: spacing after `@implementation` and before `@end`

Reply:
Done